includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1605,6 +1605,16 @@ public static function add_admin_notices() {
 	public static function get_error_details_json( $term ) {
 		$json = json_decode( $term->description, true );
 
+		if ( isset( $_GET['post'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Instead of grabbing the `$_GET['post']` can the `get_error_details_json` method rather be passed the `$post` from the `filter_tag_row_actions` method? Or rather, you could just use `get_post()` here since the `$post` global should have been set.

```suggestion
		$post = get_post();
		if ( $post instanceof WP_Post && AMP_Validated_URL_Post_Type::POST_TYPE_SLUG === $post->post_type ) {
```

Reply:
Yea that makes sense; done in https://github.com/ampproject/amp-wp/pull/5835/commits/e600804ac76efeca924700d3530ba2cf1a7eec62.